### PR TITLE
Add examples for authorization and authentication

### PR DESCRIPTION
**Pull request by nishantwrp**
This PR adds an example to set up a YugabyteDB CDC pipeline with a kafka cluster that has authentication and authorization enabled. In this example we create different users for source connectors, sink connectors and the debezium connect worker. We give these users permissions only to access the topics and groups that they need.

### CODE REVIEW

File: authorization/README.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+    ```
+
+4. Start the docker containers
+
+    ```sh
+    docker-compose up -d
+    ```
+
+5. Give the required permissions to each user
+    ```sh
```
Comment (fourpointfour) [inline authorization/README.md:146]: These commands seem like they are supposed to be executed inside some docker container, you will need to give the command to login to the container as well.

Or maybe make the whole command copy-paste worthy:
```
docker exec broker bash -c "command here"
```

File: authorization/deploy-sinks.sh [hunk 2]
```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
```
Comment (fourpointfour) [inline authorization/deploy-sinks.sh:2]: I am hoping the user doesn't need to edit anything in these scripts.

File: authorization/README.md (hunk 1 above)
Comment (nishantwrp) [reply authorization/README.md:146]: Actually, no. The user needs to download the `kafka-acls` cli and run these commands.

File: authorization/deploy-sinks.sh (hunk 2 above)
Comment (nishantwrp) [reply authorization/deploy-sinks.sh:2]: Just incorporated changes done in https://github.com/yugabyte/cdc-examples/commit/9352db2d27b6fc217ebeae67ebe419c5f384618c in this PR. User won't need to modify anything in these scripts.

**Review by fourpointfour — APPROVED** [review]

**Pull request merged**